src/main/java/mate/academy/hibernate/relations/model/Actor.java

```diff
@@ -1,8 +1,18 @@
 package mate.academy.hibernate.relations.model;
 
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+
+@Entity
 public class Actor implements Cloneable {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
     private Long id;
     private String name;
+    @ManyToOne
```

Comment:
Unidirectional relation is fine for this task, but I'd suggest exercising and playing with the various types, like bidirectional, having only 2 tables in that situation, using `mappedBy` property, `@JoinColumn`, `@JoinTable`  etc. 
Would be very beneficial